broker/applications/osb-broker/src/api-controllers/middleware/QuotaClient.js

```diff
@@ -0,0 +1,43 @@
+/* jshint ignore:start */
+'use strict';
+
+const _ = require('lodash');
+const config = require('@sf/app-config');
+const {
+  CONST,
+  HttpClient
+} = require('@sf/common-utils');
+const logger = require('@sf/logger');
+
+class QuotaClient extends HttpClient {
+  constructor(options) {
+    super(_.defaultsDeep({
+      baseUrl: config.quota_app.quota_app_url,
+      headers: {
+        Accept: 'application/json'
+      },
+      followRedirect: false
+    }, options));
+    this.username = config.quota_app.username;
+    this.password = config.quota_app.password;
+  }
+
+  async getQuotaValidStatus(options) {
+    const orgOrSubaccountId = _.get(options, 'orgOrSubaccountId');
```

Comment:
Ad we discussed earlier, do you plan to use a PUT API in case the quota calculation is dynamic ?

Reply:
Yeah, I will add code to determine which endpoint to use based on a setting in plan/service.

Reply:
ah ok...Sorry for jumping in early before you opened up for review. :)

Reply:
No issues.. I was anyways going to ask for a customary look at structure etc before starting on UTs.